website/events/templates/events/event.html

```diff
@@ -217,11 +217,8 @@ <h1 class="text-center section-title">{{ event.title }}</h1>
                                             {% endblocktrans %}
                                         {% endif %}
                                         {% if event.after_cancel_deadline and registration.queue_position == 0 %}
-                                            {% blocktrans trimmed with costs=event.fine %}
-                                                Cancellation isn't possible anymore without having to pay the full costs of €{{ costs }}.
-                                                Also note that you will be unable to re-register.{% endblocktrans %}
-                                            {% blocktrans trimmed %}
-                                                Note: If you have any COVID-19 symptoms you will not have to pay these fees. Let us know via info@thalia.nu that this is this reason for your cancellation.
+                                            {% blocktrans trimmed with no_cancellation=event.cancel_too_late_message%}
+                                                {{ no_cancellation }}
```

Comment:
Apparently this is not working, since the tests are failing on this text not showing up.

Reply:
There is no translation here anymore